climatic/cli/Linux.py

```diff
@@ -83,3 +84,55 @@ def logout(self):
         """ Logout from CLI interface.
         """
         self.connection.terminal.sendline('exit')
+
+
+####################################################################################################
+## Ser2NetLinux
+
+class Ser2NetLinux(SshLinux):
```

Comment:
Why is it extendind the SshLinux instead of Linux? Is it only for the implementation of the logout function? If so, move it to the Linux class and extend the Ser2NetLinux from Linux, because the SshLinux may have invalid methods for Ser2NetLinux in the future.